netcon/mtclient/mtclient.cpp

```diff
@@ -1351,8 +1348,10 @@ int MainMultiplayerMenu() {
           char *pcount = strchr(tokp, ' ');
           // if(pcount)
           {
-            if (!pcount)
-              pcount = "";
+            if (!pcount) {
+              static char quietWarning[5] = "";
```

Comment:
this fix does not seem optimal. How about simply `pcount = (char *)"";` ? My compiler does not warn when doing that

Reply:
While `pcount = (char *)"";` does quiet the warning, the line under it will try to modify it. On most systems, strings like that are constant and it might crash the app.
An alternative is to not modify the string if `pcount` is nil.

Reply:
The alternative of not setting the string when pcount is nullptr seems better in this context